packaging/rpm/telemetry-agent.spec

```diff
@@ -73,42 +76,60 @@ if [ ! -d /run/percona-telemetry-agent ]; then
     install -m 0755 -d -oroot -groot /run/percona-telemetry-agent
 fi
 # Create new linux group
-groupadd percona-telemetry
 # For telemetry-agent to be able to read/remove the metric files
-usermod -a -G percona-telemetry daemon
+/usr/bin/getent group percona-telemetry || groupadd percona-telemetry >/dev/null 2>&1 || :
+usermod -a -G percona-telemetry daemon >/dev/null 2>&1 || :
 
 %post -n percona-telemetry-agent
-%systemd_post percona-telemetry-agent.service
-/usr/bin/systemctl enable percona-telemetry-agent >/dev/null 2>&1 || :
-/usr/bin/systemctl start percona-telemetry-agent
+chown -R daemon:percona-telemetry %{_log_dir} >/dev/null 2>&1 || :
+chmod g+w %{_log_dir}
+# Move the old logfiles, if present during update
+if [ -e /var/log/percona/telemetry-agent.log ] && [ -e /var/log/percona/telemetry-agent-error.log ]; then
+    chmod 0775  %{_log_dir}
+    mv /var/log/percona/telemetry-agent.log /var/log/percona/telemetry-agent/
```

Comment:
btw, what with the rest of logs that were already rotated and have suffixes like `.log.1` ?

Reply:
Yes, good point, thanks. I've updated the changes to move the logrotated files as well.